src/bandwidthRtc.ts

```diff
@@ -28,7 +31,7 @@ class BandwidthRtc {
    * @param authParams connection credentials
    * @param options additional connection options; usually unnecessary
    */
-  async connect(authParams: RtcAuthParams, options?: RtcOptions) {
+  async connect(authParams: RtcAuthParams, options?: RtcOptions): Promise<BandwidthRtcV2 | BandwidthRtcV3> {
```

Comment:
Returning the delegate here is key and allows us to override functions using the WebRTC primitives with the SDK being none the wiser.

Reply:
Should we include some info in the function doc about a Promise being returned?

Reply:
Possibly... it's a bit of a hack to work around the shim; eventually the trogdor test will be using the "delegate" directly and will be able to override functions right after constructing an instance of the SDK. In other words this is hopefully temporary and hopefully never going to be used by a customer, so I have mixed feelings.

Reply:
I see now that it is/was async, so the promise is implied